tensorboard/plugins/hparams/api.py

```diff
@@ -14,7 +14,85 @@
 # ==============================================================================
 """Experimental public APIs for the HParams plugin.
 
-These are porcelain on top of `api_pb2` (`api.proto`) and `summary.py`.
+This module supports a spectrum of use cases, depending on how much
+structure you want. In the simplest case, you can simply collect your
+hparams into a dict, and use a Keras callback to record them:
+
+>>> from tensorboard.plugins.hparams import api as hp
+>>> hparams = {
+...     "optimizer": "adam",
+...     "fc_dropout": 0.2,
+...     "neurons": 128,
+...     # ...
+... }
+>>>
+>>> model = model_fn(hparams)
+>>> callbacks = [
+>>>     tf.keras.callbacks.TensorBoard(logdir),
+>>>     hp.KerasCallback(logdir, hparams),
+>>> ]
+>>> model.fit(..., callbacks=callbacks)
+
+To control how hyperparameters and metrics appear in the TensorBoard
+frontend, you can define `HParam` and `Metric` objects and collect them
```

Comment:
frontend -> ui (or omit)

The doc says `...The domains will also be reflected in the TensorBoard UI.` below.

Reply:
Good point. Consistency is good. Will use “UI”.